fbw-a32nx/src/wasm/fbw_a320/build.sh

```diff
@@ -79,6 +79,12 @@ clang++ \
   -fno-exceptions \
   -fms-extensions \
   -fvisibility=hidden \
+  -fdata-sections \
+  -fno-stack-protector \
+  -fstack-size-section \
+  -fwritable-strings \
```

Comment:
```suggestion
```
I don't think this flag is a good idea. It's to allow old code that expected to be able to write to string constants.